src/OpenDebugAD7/Tracepoint.cs

```diff
@@ -94,10 +93,9 @@ private int GetInterpolatedLogMessage(string logMessage, IDebugThread2 pThread,
                     else
                     {
                         string toInterpolate = keyValuePair.Value;
-                        hr = InterpolateVariable(toInterpolate.Substring(1, toInterpolate.Length - 2), topFrame[0].m_pFrame, radix, out value);
-                        if (hr < 0)
+                        if (InterpolateVariable(toInterpolate.Substring(1, toInterpolate.Length - 2), topFrame[0].m_pFrame, radix, out value) < 0)
```

Comment:
why not make this return value the HR? and then ...?

Reply:
I did not want to stop the log message from interpolating half way through. This allows it to completly interpolate the expressions and indicate an error has occured for the ones that failed (if there are multiple)